scripts/mirna_quantification.py

```diff
@@ -285,16 +285,18 @@ def collapsed_contribution(aln: pysam.AlignedSegment) -> float:
             collapsed = float(coll.group())
 
     except AttributeError:
-        sys.stdout.write(f"Invalid query name: '{aln.query_name}'.\n" +
-                         "Option --collapsed specified but query name does" +
-                         " not include the number of collapsed sequences.\n" +
-                         "Check SAM file consistency and CLI options" +
-                         " --collapsed and --nh.\n")
+        sys.stdout.write(
+            f"Invalid query name: '{aln.query_name}'.\n" +
+            "Option --collapsed specified but query name does " +
+            "not include the number of collapsed sequences.\n" +
+            "Check SAM file consistency and CLI options" +
+            " --collapsed and --nh.\n"
+        )
```

Comment:
What's with the all the `+`es? Better to use parantheses for multiline strings. Or triple quoted strings. Weird that none of the linters complain here.